invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -62,7 +62,7 @@ class CommunityCard extends Component {
         />
         <Card.Content>
           <Card.Header>
-            {_truncate(community.metadata.title, { length: 50 })}
+            {_truncate(community.metadata.title, { length: 30 })}
```

Comment:
With length 50, big names were being rendered  outside of the card. In mobile 30 seems to be good enough as well.